parcellab/common/templates/_pod.tpl

```diff
@@ -158,6 +149,13 @@ spec:
             name: {{ $name }}
         {{- end }}
       {{- end }}
+  {{- if $extraContainers }}
+  {{- range $extraContainers }}
+    {{- include "common.container"
```

Comment:
Same as above

```suggestion
    - {{- include "common.container"
```

Therefore you do not limit it in the implementation of the container.